CloudEventify/ITypeMap.cs

```diff
@@ -1,6 +1,6 @@
 namespace CloudEventify;
 
```

Comment:
Whats the difference between this interface and IDictionary<T,T>, doesn't it have the same method?

Reply:
I cannot change this. it is not a back and forth map (it has 2 different mappings).. this is existing code to which I only renamed the object..